cmd/dra-example-controller/gpu.go

```diff
@@ -59,6 +59,35 @@ func (g *gpudriver) Allocate(crd *nascrd.NodeAllocationState, claim *resourcev1.
 	return onSuccess, nil
 }
 
+func (g *gpudriver) AllocateImmediately(crd *nascrd.NodeAllocationState, ca *controller.ClaimAllocation) error {
```

Comment:
If we call this one `AllocateImmediately` can we call the other one `AllocateDelayed` for consistency? Also, is there a reason to not keep the function signatures identical (possibly with the only difference being the removal of `selectedNode`?

Reply:
Sure, but can we at least wrap claim, claimParams, class and classParams into ClaimAllocation right in the beginning of Allocate and pass it into allocateDelayedClaim and allocateImmedateClaim, so we don't have 160 chars long parameters line in func definition?

Reply:
No it has to be broken down again before being passed into gpu module, never mind, I'm just underslept, sorry.

Reply:
Done.